sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -876,6 +884,20 @@ def reset_module(self, reset = False):
         else:
             return True
 
+    def reset(self):
+        """
+        Reset SFP and return all user module settings to their default state.
+
+        Returns:
+            A boolean, True if successful, False if not
+        """
+        if self.reset_module(True):
+            for retries in range(5):
+                time.sleep(1)
```

Comment:
After reset the management init time required is 2 seconds

Reply:
Sure, will do